flink-libraries/flink-table/src/main/scala/org/apache/flink/table/codegen/AggregationCodeGenerator.scala

```diff
@@ -405,6 +481,17 @@ class AggregationCodeGenerator(
       }
     }
 
+    val aggFuncCode = Seq(
+      genSetAggregationResults,
+      genAccumulate,
+      genRetract,
+      genCreateAccumulators,
+      genSetForwardedFields,
+      genSetConstantFlags,
+      genCreateOutputRow,
+      genMergeAccumulatorsPair,
+      genResetAccumulator).mkString("\n")
```

Comment:
It make sense，I have look at ProcessFunctionWithCleanupState，the cleanUp should be called whenever cleanupState is called.